src/test/scala/viper/gobra/reporting/StatsCollectorTests.scala

```diff
@@ -0,0 +1,147 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2011-2020 ETH Zurich.
+package viper.gobra.reporting
+
+import org.rogach.scallop.throwError
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.funsuite.AnyFunSuite
+import viper.gobra.ast.frontend._
+import viper.gobra.frontend.info.base.Type
+import viper.gobra.frontend.{Config, PackageEntry, ScallopGobraConfig}
+import viper.gobra.util.{DefaultGobraExecutionContext, GobraExecutionContext}
+import viper.gobra.Gobra
+
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+
+class StatsCollectorTests extends AnyFunSuite with BeforeAndAfterAll {
+  val statsCollectorTestPathPropertyName = "GOBRATESTS_REGRESSIONS_DIR"
+  val statsCollectorTestDir: String = System.getProperty(statsCollectorTestPathPropertyName, "src/test/resources/stats_collector")
```

Comment:
You can directly set the path. We only do the indirection via properties for tests that are meant to be called from external and thus should be configurable